WordPressKit/PostServiceRemoteREST.m

```diff
@@ -435,6 +438,8 @@ - (NSDictionary *)parametersWithRemotePost:(RemotePost *)post
     }
     parameters[@"featured_image"] = post.postThumbnailID ? [post.postThumbnailID stringValue] : @"";
     parameters[@"metadata"] = [self metadataForPost:post];
+    
+    parameters[@"sticky"] = post.isStickyPost ? @"1" : @"0";
```

Comment:
I'm not %100 sure if the endpoint will accept string values of "1" and "0" correctly.  I did a quick test at `https://developer.wordpress.com/docs/api/console/` and tried creating new posts. The endpoint interprets a numeric 1 as true, numeric 0 as false, string "true" as true and string "false" as false, but a string "1" is not evaluated to true... it evaluated to false (if all that makes sense). 

Maybe double check and if you get the same results as I did we might change this line accordingly.